### PR TITLE
ci(oci): push helm chart package to github container registry

### PR DESCRIPTION
**Pull request by mwangggg**
fixes: #161

### CODE REVIEW

**Comment by mwangggg** [comment]
test run in forked repo: https://github.com/mwangggg/cryostat-helm/actions/runs/9964046130

File: .github/workflows/push-oci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          CR_SKIP_EXISTING: "true"
+        with:
+          charts_dir: charts
+          config: .github/helm-release-config.yml
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (andrewazores) [inline .github/workflows/push-oci.yml:44]:
```suggestion
          password: ${{ secrets.GH_PKGS_WRITE }}
```

**Review by andrewazores — APPROVED** [review]

**Comment by andrewazores** [comment]
The [release](https://github.com/cryostatio/cryostat-helm/releases/tag/v2.0.0-dev) got created by [the workflow run](https://github.com/cryostatio/cryostat-helm/actions/runs/9980032995) with no apparent errors, but [no package](https://github.com/orgs/cryostatio/packages?repo_name=cryostat-helm). Maybe it just silently failed?

**Comment by andrewazores** [comment]
```
$ podman pull ghcr.io/cryostatio/cryostat-helm/cryostat:2.0.0-dev
Trying to pull ghcr.io/cryostatio/cryostat-helm/cryostat:2.0.0-dev...
Error: parsing image configuration: unsupported image-specific operation on artifact with type "application/vnd.cncf.helm.config.v1+json"
```

**Comment by andrewazores** [comment]
image

Image name looks wrong...

**Comment by andrewazores** [comment]
Ah I see the problem, the package does exist but it has internal visibility. I'll figure out how to update this. I think the CI job is actually working properly.

**Comment by tthvo** [comment]
Also, I think the release should not be created? Just the container image needs pushing to `ghcr.io`?

https://github.com/helm/chart-releaser-action?tab=readme-ov-file#inputs

The `skip_upload` can be set to avoid creating releases.

**Comment by andrewazores** [comment]
Looks like a release still got created. The `chart_version` output is also not what I expected it to be.

**Comment by tthvo** [comment]
Hmm, seems to be the image digest. Though, according this helm docs: https://helm.sh/docs/topics/registries/#the-push-subcommand, only `registry/namespace` can be specified with `helm push`.

**Pull request merged**